SonarQube.Bootstrapper/BuildAgentUpdater.cs

```diff
@@ -13,27 +13,13 @@
 
 namespace SonarQube.Bootstrapper
 {
-    public class BuildAgentUpdater
+    public class BuildAgentUpdater : IBuildAgentUpdater
     {
-        private const string SonarQubeIntegrationFilename = "SonarQube.MSBuild.Runner.Implementation.zip";
-        private const string IntegrationUrlFormat = "{0}/static/csharp/" + SonarQubeIntegrationFilename;
```

Comment:
I've tried to collect all of the constants that affect the "logical bootstrapper API" into the BootStrapperSettings class, so these constants have been moved there.